oncopipe/oncopipe/__init__.py

```diff
@@ -207,24 +192,80 @@ def coerce_namedlist_to_string(obj):
     else:
         dest_dir, dest_file = os.path.split(dest)
     os.makedirs(dest_dir, exist_ok=True)
```

Comment:
While doing some testing, I realized that this function call will fail if the given destination path is just a file name (_i.e._ referring to the current working directory). An easy solution is converting the `dest_dir` to an absolute path right before passing it to the function. I confirmed that running `os.path.abspath()` on an empty string returns the current working directory:

```
>>> dest_dir = ""
>>> os.path.abspath(dest_dir)
'/Users/bgrande/Desktop/testdir'
```

```suggestion
    os.makedirs(os.path.abspath(dest_dir), exist_ok=True)
```

Reply:
Good catch!